<ui version="4.0" >
 <class>SelfTestDialog</class>
 <widget class="QWidget" name="SelfTestDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>An error occurred during the startup of the Akonadi server. The following self-tests are suposed to help with tracking down and solving this problem. When requesting support or reporting bugs, please always include this report.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="testView" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="headerHidden" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="detailsGroup" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLabel" name="detailsLabel" >
        <property name="text" >
         <string/>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags" >
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
